internal/provider/template.go

```diff
@@ -232,9 +232,7 @@ description: |-
 {{ printf "{{tffile %q}}" .ExampleFile }}
 {{- end }}
 
-{{ .SchemaMarkdown | trimspace }}
-
-{{ if .HasImport -}}
+{{ .SchemaMarkdown | trimspace }}{{ if .HasImport }}
```

Comment:
```suggestion
{{ .SchemaMarkdown | trimspace }}
{{- if .HasImport }}

```
Switching to this will continue to preserve the spacing above the import header (we don't have tests for this area unfortunately, but I believe that was the original intention)

Reply:
Hey @austinvalle! Thanks for the heads up! 
I've run the terraform-plugin-docs code against our codebase. I'm afraid that's not true 😢 The single line whitespace between `trimspace` and `if .HasImport` generates a redundant extra whitespace line. 
I guess what you might want to do might be:
```suggestion
{{ .SchemaMarkdown | trimspace }}{{ if .HasImport }}

```
this, maybe? 🤔 The `if .HasImport` stays at the same line with `trimspace`, but we preserve the spacing between the import header and the Schema Markdown.

Reply:
Oh, that's absolutely my bad, I overlooked the `-` preceding the `if` statement, sorry for taking your time. 
I tested the code you suggested, and works perfect 👍. 
@YanniHu1996 is working GMT+8, he would include the change into this PR tomorrow. 
Thanks again for the suggestion, and thanks to @YanniHu1996 for the PR.

Reply:
No problem at all! I'll keep an eye out for that update 👍🏻